### ISSUE TITLE
issue: try add to context menu failed

### ISSUE THREAD

**Issue by tommynanny** [Decimation/SmartImage#7]
![image](https://user-images.githubusercontent.com/22390882/93192840-19a6f480-f714-11ea-9a89-6f1ab8ca9318.png)

**Comment by Decimation** [comment]
Did you get a UAC alert to allow running the Command Prompt?

**Comment by tommynanny** [comment]
No UAC alert shown when starting the program or after I select number 3 from the menu. I also tried to run as admin, but it throws the same error.

**Comment by Decimation** [comment]
I'll take a look at it when possible!

**Comment by Decimation** [comment]
Can you try 1.9.4 and attach the generated crash log? I added crash handling which might help identify the problem. It would also be very helpful if you can give me more information such as what you were searching.

**Comment by tommynanny** [comment]
With the 1.9.4
![image](https://user-images.githubusercontent.com/22390882/93847920-4c9e3a80-fc76-11ea-9cea-d30b5578e8ca.png)

here is the crashed log generated on the desktop:
`----------Exception----------
Exception message: Exception of type 'SmartImage.Utilities.SmartImageException' was thrown.
Exception stack trace:    at SmartImage.RuntimeInfo.get_IsContextMenuAdded()
   at SmartImage.Commands.<>c.<RunCommandMenu>b__9_3()
   at SmartImage.Commands.HandleConsoleOptions(ConsoleOption[] options, Boolean multiple)
   at SmartImage.Commands.RunCommandMenu()
   at SmartImage.Program.Main(String[] args)
Source: SmartImage
HResult: -2146233088
Site: Boolean get_IsContextMenuAdded()

----------Config----------
Search engines: All
Priority engines: SauceNao
SauceNao authentication: × (Basic)
Imgur authentication: ×
Image upload service: ImgOps
Config location: C:\Users\tommy\Downloads\SmartImage.cfg

----------Program Info----------
Version: 1.9.4.0`

All I did is start the program and press [3] "[3]: Add/remove context menu integration" and the error occurred.

**Comment by Decimation** [comment]
`IsContextMenuAdded` runs the command `reg query HKEY_CLASSES_ROOT\*\shell\SmartImage\command` and checks for its output. It should return `true` if the output is something like this:

```
HKEY_CLASSES_ROOT\*\shell\SmartImage\command
    (Default)    REG_SZ    C:\Users\Deci\Desktop\SmartImage.exe "%1"
```

and `false` if it returns:

```
ERROR: The system was unable to find the specified registry key or value.

```

Could you run the command `reg query HKEY_CLASSES_ROOT\*\shell\SmartImage\command` manually and tell me what it gives you?

**Comment by tommynanny** [comment]
the cmd gives me the second one which is no registry key found
![image](https://user-images.githubusercontent.com/22390882/93848970-fbdc1100-fc78-11ea-9798-2a0e304d01cb.png)

**Comment by Decimation** [comment]
It gives you this exact output?

```
ERROR: The system was unable to find the specified registry key or value.

```

**Comment by tommynanny** [comment]
okay, now it works now. You gave me the hint that it might be my window 10 system language not being English. It works like charm, and I have just switch my lanauge back. This is such a great tool! :D
![image](https://user-images.githubusercontent.com/22390882/93850073-5aa28a00-fc7b-11ea-87a1-879bd2fdac30.png)

(I realized under my original language setting, the cmd window did not pop up when I selected [3])
